### PR TITLE
feat: Example/express

### PR DESCRIPTION
**Pull request by iresharma**
## Summary of change

(A few sentences about this PR)

## Related issues

-   Link to issue1 here
-   Link to issue1 here

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

## Documentation changes

(If relevant, please create a PR in our [docs repo](https://github.com/supertokens/docs), or create a checklist here highlighting the necessary changes)

## Checklist for important updates

-   [ ] Changelog has been updated
-   [ ] `coreDriverInterfaceSupported.json` file has been updated (if needed)
    -   Along with the associated array in `lib/ts/version.ts`
-   [ ] `frontendDriverInterfaceSupported.json` file has been updated (if needed)
-   [ ] Changes to the version if needed
    -   In `package.json`
    -   In `package-lock.json`
    -   In `lib/ts/version.ts`
-   [ ] Had run `npm run build-pretty`
-   [ ] Had installed and ran the pre-commit hook
-   [ ] Issue this PR against the latest non released version branch.
    -   To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
    -   If no such branch exists, then create one from the latest released branch.
-   [ ] If have added a new web framework, update the `add-ts-no-check.js` file to include that
-   [ ] If added a new recipe / api interface, then make sure that the implementation of it uses NON arrow functions only (like `someFunc: function () {..}`).
-   [ ] If added a new recipe, then make sure to expose it inside the recipe folder present in the root of this repo. We also need to expose its types.

## Remaining TODOs for this PR

-   [ ] Item1
-   [ ] Item2

### CODE REVIEW

File: examples/express/with-emailpassword/package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+    "name": "with-emailpassword",
+    "version": "0.0.1",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "start:frontend": "cd frontend && npm run start",
```
Comment (rishabhpoddar) [inline examples/express/with-emailpassword/package.json:7]: add a readme in the with-emailpassword folder.

File: examples/express/with-emailpassword/backend/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    "dependencies": {
+        "cors": "^2.8.5",
+        "express": "^4.18.1",
+        "helmet": "^5.1.0",
+        "morgan": "^1.10.0",
+        "npm-run-all": "^4.1.5",
+        "supertokens-node": "latest",
+        "ts-node-dev": "^2.0.0",
+        "typescript": "^4.7.2"
```
Comment (rishabhpoddar) [inline examples/express/with-emailpassword/backend/package.json:18]: this should be empty cause all these should be in the package.json of the with-emailpassword folder.

File: examples/express/with-emailpassword/frontend/package.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        "@types/react": "^18.0.18",
+        "@types/react-dom": "^18.0.6",
+        "axios": "^0.21.0",
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0",
+        "react-router-dom": "^6.2.1",
+        "react-scripts": "5.0.1",
+        "supertokens-auth-react": "latest",
+        "typescript": "^4.8.2",
+        "web-vitals": "^2.1.4"
```
Comment (rishabhpoddar) [inline examples/express/with-emailpassword/frontend/package.json:20]: this should be empty cause all these should be in the package.json of the with-emailpassword folder.

File: examples/express/with-emailpassword/backend/package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    "scripts": {
+        "start": "npx ts-node-dev --project ./tsconfig.json ./index.ts"
+    },
+    "dependencies": {
+        "cors": "^2.8.5",
+        "express": "^4.18.1",
+        "helmet": "^5.1.0",
+        "morgan": "^1.10.0",
+        "npm-run-all": "^4.1.5",
+        "supertokens-node": "latest",
```
Comment (rishabhpoddar) [inline examples/express/with-emailpassword/backend/package.json:16]: The supertokens-node in the example app should import from this repo itself. Can you make it so that this dependency points to the code in this repo? Or else you can even remove this dependency and in the app, import like "../../../" or something.

File: test/exampleTestHelpers.js [hunk 5]
```diff
@@ -0,0 +1,147 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
```
Comment (rishabhpoddar) [inline test/exampleTestHelpers.js:4]: - why is this here? It should go in the examples foler.
- you also need to add a github workflow to run the tests for all the example apps on each PR creation (see how it's done in auth-react repo)

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: test/exampleTestHelpers.js (hunk 5 above)
Comment (iresharma) [reply test/exampleTestHelpers.js:4]: > why is this here? It should go in the examples foler.
This is how it is auth-react repo

**Review by rishabhpoddar — APPROVED** [review]
